doc/dependency_decisions.yml

```diff
@@ -577,3 +423,171 @@
     :versions:
     - 1.5.1
     :when: 2018-01-17 12:06:07.468492000 Z
+- - :whitelist
+  - MIT/X11
+  - :who:
+    :why:
+    :versions: []
+    :when: 2018-10-08 14:41:10.872704000 Z
+- - :whitelist
+  - BSD-like
+  - :who:
+    :why:
+    :versions: []
+    :when: 2018-10-08 14:41:20.655159000 Z
+- - :whitelist
+  - CC-BY-4.0
+  - :who:
+    :why:
+    :versions: []
+    :when: 2018-10-08 14:41:28.474576000 Z
+- - :whitelist
+  - WTFPL
+  - :who:
+    :why:
+    :versions: []
+    :when: 2018-10-08 14:41:53.058253000 Z
+- - :whitelist
+  - CC0-1.0
+  - :who:
+    :why:
+    :versions: []
+    :when: 2018-10-08 14:42:01.551501000 Z
+- - :whitelist
+  - BSD-new
+  - :who:
+    :why:
+    :versions: []
+    :when: 2018-10-08 14:42:59.682695000 Z
+- - :whitelist
+  - CC-BY-3.0
+  - :who:
+    :why:
+    :versions: []
+    :when: 2018-10-08 14:43:09.580575000 Z
+- - :approve
+  - abab
+  - :who: Didier Di Cesare
+    :why: W3C 3-clause BSD License
+    :versions: []
+    :when: 2018-10-08 14:48:54.145370000 Z
+- - :approve
+  - atob
+  - :who: Didier Di Cesare
+    :why: MIT OR Apache-2.0
+    :versions: []
+    :when: 2018-10-08 14:49:24.617321000 Z
+- - :approve
+  - esutils
+  - :who: Didier Di Cesare
+    :why: BSD 2-Clause Simplified License
+    :versions: []
+    :when: 2018-10-08 14:51:56.810452000 Z
+- - :approve
+  - node-forge
+  - :who: Didier Di Cesare
+    :why: BSD-3-Clause OR GPL-2.0
+    :versions: []
+    :when: 2018-10-08 14:52:26.815935000 Z
+- - :approve
+  - pako
+  - :who: Didier Di Cesare
+    :why: MIT AND Zlib
+    :versions: []
+    :when: 2018-10-08 14:52:46.864375000 Z
+- - :approve
+  - stackframe
+  - :who: Didier Di Cesare
+    :why: MIT License
+    :versions: []
+    :when: 2018-10-08 14:53:35.926686000 Z
+- - :approve
+  - sha.js
+  - :who: Didier Di Cesare
+    :why: MIT AND BSD-3-Clause
+    :versions: []
+    :when: 2018-10-08 14:54:42.187406000 Z
+- - :approve
+  - rgb
+  - :who: Didier Di Cesare
+    :why: MIT License
+    :versions: []
+    :when: 2018-10-08 14:55:01.288430000 Z
+- - :approve
+  - buffers
+  - :who: Didier Di Cesare
+    :why: MIT/X11 License
+    :versions: []
+    :when: 2018-10-08 14:58:43.955104000 Z
+- - :approve
+  - cyclist
+  - :who: Didier Di Cesare
+    :why: MIT License
+    :versions: []
+    :when: 2018-10-08 14:59:16.150521000 Z
+- - :approve
+  - domelementtype
+  - :who: Didier Di Cesare
+    :why: BSD 2-Clause Simplified License
+    :versions: []
+    :when: 2018-10-08 15:00:45.031274000 Z
+- - :approve
+  - domutils
+  - :who: Didier Di Cesare
+    :why: BSD 2-Clause Simplified License
+    :versions: []
+    :when: 2018-10-08 15:01:15.851586000 Z
+- - :approve
+  - isnumeric
+  - :who: Didier Di Cesare
+    :why: MIT License
+    :versions: []
+    :when: 2018-10-08 15:02:28.386300000 Z
+- - :approve
+  - node-fs
```

Comment:
Is this runtime dependency? Isn't it used just for compilation? Then we should probably ignore dev group of npm packages.